cicecore/cicedynB/general/ice_flux.F90

```diff
@@ -738,11 +738,19 @@ subroutine init_coupler_flux
       ! fluxes received from ocean
       !-----------------------------------------------------------------
 
-      ss_tltx(:,:,:)= c0              ! sea surface tilt (m/m)
-      ss_tlty(:,:,:)= c0
-      uocn  (:,:,:) = c0              ! surface ocean currents (m/s)
-      vocn  (:,:,:) = c0
-      frzmlt(:,:,:) = c0              ! freezing/melting potential (W/m^2)
+      ss_tltx(:,:,:) = c0              ! sea surface tilt (m/m)
```

Comment:
the order of these lines is nonintuitive...  suggest putting ss_tlt* together, etc